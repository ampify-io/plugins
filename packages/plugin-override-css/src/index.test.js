import { prepare, run } from './';

beforeAll(() => {
  document.documentElement.innerHTML = `<html lang="en"><body></body></html>`;
});

test('prepare', () => {
  prepare({
    rules: [
      'div { color: red}',
      'body { background: blue }',
      '.test { font-family: serif}',
      '#header { margin: 10px}'
    ]
  });

  expect(document.documentElement.outerHTML).toMatchSnapshot();
});

test('run', () => {});
