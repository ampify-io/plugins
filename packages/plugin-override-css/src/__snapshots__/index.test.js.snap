// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prepare 1`] = `
"<html><head><style>div {
  color: red;
}
body {
  background: blue;
}
.test {
  font-family: serif;
}
#header {
  margin: 10px;
}
</style></head><body></body></html>"
`;
